locallm-images.md

```diff
@@ -1,17 +1,19 @@
-## Images currently built from GH Actions in this repository
+## Images (x86_64, aarch64) currently built from GH Actions in this repository
 
 - quay.io/redhat-et/locallm-model-service:latest
 - quay.io/redhat-et/locallm-text-summarizer:latest
 - quay.io/redhat-et/locallm-chatbot:latest
 - quay.io/redhat-et/locallm-rag:latest
 - quay.io/redhat-et/locallm-codegen:latest
 - quay.io/redhat-et/locallm-chromadb:latest
+- quay.io/redhat-et/locallm-object-detection-client:latest
+- quay.io/redhat-et/locallm-object-detection-server:latest
 
-## Model Images currently in `quay.io/redhat-et/locallm-models`
+## Model Images (x86_64, aarch64) currently in `quay.io/redhat-et/locallm-*`
 
-- quay.io/redhat-et/locallm-models:llama-2-7b-chat.Q5_K_S.gguf
+- quay.io/redhat-et/locallm-llama-2.7b-gguf:latest
```

Comment:
Is this right @sallyom? with the period between `llama-2` and `7b`?

Reply:
yes, unfortunately this is how they were pushed - it is ugly - i'll update them & create new quay repos now.

Reply:
👍  was just making sure it wasn't a typo

Reply:
updated & pushed